core/api/src/schema/epoch.rs

```diff
@@ -1,26 +1,46 @@
 use crate::schema::{Address, Hash, MerkleRoot, Uint64};
 
 #[derive(GraphQLObject, Clone)]
-#[graphql(description = "Epoch")]
+#[graphql(
+    description = "Epoch is a single digital record created within a blockchain. \
+                   Each epoch contains a record of the previous Epoch, \
+                   and when linked together these become the “chain”.\
+                   An epoch is always composed of header and body."
+)]
 pub struct Epoch {
-    header:            EpochHeader,
+    #[graphql(description = "The header section of an epoch")]
+    header: EpochHeader,
+    #[graphql(description = "The body section of an epoch")]
     ordered_tx_hashes: Vec<Hash>,
 }
 
 #[derive(GraphQLObject, Clone)]
-#[graphql(description = "Epoch header")]
+#[graphql(description = "An epoch header is like the metadata of an epoch.")]
 pub struct EpochHeader {
-    pub chain_id:     Hash,
-    pub epoch_id:     Uint64,
-    pub pre_hash:     Hash,
-    pub timestamp:    Uint64,
-    pub order_root:   MerkleRoot,
+    #[graphql(
+        description = "Identifier of a chain in order to prevent replay attacks across channels "
+    )]
+    pub chain_id: Hash,
+    #[graphql(description = "Known as the block height like other blockchain")]
+    pub epoch_id: Uint64,
+    #[graphql(description = "The merkle root of the previous epoch")]
```

Comment:
```suggestion
    #[graphql(description = "Previous epoch hash")]
```

This is a hash, not a mercke root.

Reply:
OK, I have make the description more clearly now